api-tests/src/steps/ipv-steps.ts

```diff
@@ -261,3 +285,14 @@ Then(
     );
   },
 );
+
+Then(
+  "I get a {string} MFA reset result",
```

Comment:
Can this ever be anything other than `successful`? Might be worth making a similar change to above...